chat-plugins/chat-monitor.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+
+const FS = require('../lib/fs');
+
+const MONITOR_FILE = 'config/chat-plugins/chat-monitor.json';
+
+/** @type {{[k: string]: string[]}?} */
+let data = null;
+try {
+	data = require(`../${MONITOR_FILE}`);
+} catch (e) {
+	if (e.code !== 'MODULE_NOT_FOUND' && e.code !== 'ENOENT') throw e;
+}
+/** @type {{[k: string]: string[]}} */
+const filterWords = !data || typeof data !== 'object' ? {publicfilter: [], filter: [], autolock: [], namefilter: []} : data;
```

Comment:
> `|| typeof data !== 'object'`

It's not like you're checking to see if `data` matches the spec. Just let it go.